src/main/java/com/ericsson/ei/subscriptionhandler/SpringRestTemplate.java

```diff
@@ -49,12 +54,20 @@ public SpringRestTemplate(RestTemplateBuilder builder) {
      * @param headerContentMediaType
      * @return integer
      */
-    public int postDataMultiValue(String notificationMeta, MultiValueMap<String, String> mapNotificationMessage, String headerContentMediaType) {
+    public int postDataMultiValue(String notificationMeta, MultiValueMap<String, String> mapNotificationMessage, String headerContentMediaType, String ...args) {
         ResponseEntity<JsonNode> response;
+        
         try {
             HttpHeaders headers = new HttpHeaders();
             headers.setContentType(MediaType.valueOf(headerContentMediaType));
             if (headerContentMediaType.equals(MediaType.APPLICATION_FORM_URLENCODED.toString())) { //"application/x-www-form-urlencoded"
+                
+                if(notificationMeta.contains("jenkins") && args.length != 0) { 
```

Comment:
There does not need to be any Jenkins specific code. Or neither any other application.

Reply:
Ok...done